src/parser/jobs/gnb/modules/OGCDDowntime.tsx

```diff
@@ -1,9 +1,20 @@
 import ACTIONS from 'data/ACTIONS'
 import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
 
+const FIRSTUSEOFFSETNOMERCY = 10000
```

Comment:
These should be split with underscores for readability (e.g. `FIRST_USE_OFFSET_NO_MERCY` and `FIRST_USE_OFFSET_BLOODFEST`. Same thing goes for the consts in the PLD module as well.

Reply:
Fixed in current version.